### ISSUE TITLE
Support Github emoji.

### ISSUE THREAD

**Issue by btholt** [adamburmister/gitprint.com#40]
[Here's](https://gitprint.com/btholt/es6-react-pres/blob/master/README.md) an example of that. It'd be a fun feature.

Thanks for this, by the way. I use it quite a bit.

**Comment by adamburmister** [comment]
Yeah... good suggestion! Sounds fun too. I'll add it to the next release milestone.

**Comment by alexandertuna** [comment]
ping

**Comment by leonoverweel** [comment]
Is this going to happen? Right now, it's a reason for me to not use gitprint.
